content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,139 @@
+      tag: 'Documentation'
+      text: 'Use CloudWatch metric streams with Kinesis Data Firehose'
+      
+---
+
+## Overview
+
+This guide provides an overview of the process for linking an AWS account with Datadog using Datadog's CloudFormation template. 
+
+At a high level, this involves creating a read-only IAM role and associated policy for Datadog's AWS account and deploying the [Datadog Forwarder][1] Lambda function. The forwarder then simplifies the process of sending logs, custom metrics, and traces to Datadog. Using the CloudFormation template provides all the tools needed to send this data to your Datadog account, and Datadog maintains the CloudFormation template to provide the latest functionality.
+
+This process can be repeated for as many AWS accounts as necessary, and you can also use the [API][2] or [Terraform][3] to set up multiple accounts at once.
+
+## Prerequisites
+
+Before getting started, ensure you have the following prerequisites:
+
+1. [AWS][4] account
+2. A Datadog account and [API key][5]. If you need a Datadog account, [sign up for a free trial][6].
+
+## Setup
+
+1. From the AWS tile on the [Integrations Page][7] in your Datadog account. Select the CloudFormation Template option:
+{{< img src="getting_started/integrations/integration-tile-setup.png" alt="Datadog integrations AWS tile">}}
+
+2. This opens the AWS Console and loads the CloudFormation stack. For this demonstration, the API key has been entered directly into the `DdApiKey` field, but you can also store this value in [Secrets Manager][8] and specify the ARN of the secret:
+{{< img src="getting_started/integrations/cloudformation-options-1.png" alt="First part of CloudFormation options for creating Datadog stacks">}}
+
+3. Ensure that the the correct [Datadog site][9] parameter is used for the `DdSite` parameter to match the region of your Datadog account. Check the required boxes from AWS and click `Create stack`:
+{{< img src="getting_started/integrations/cloudformation-options-2.png" alt="Second part of CloudFormation options for creating Datadog stacks">}}
+
+
+This begins the creation process for the Datadog stack along with three nested stacks:
+{{< img src="getting_started/integrations/cloudformation-stacks-complete.png" alt="Datadog stacks completed in CloudFormation">}}
+
+4. From the integration tile on the Datadog side, enter the AWS account ID and AWS Role name. If the role name wasn't changed in the CloudFormation setup page, use `DatadogIntegrationRole`. Datadog automatically queries AWS with the provided values, and a successful query returns the following status:
+{{< img src="getting_started/integrations/integration-tile-status.png" alt="Datadog AWS integration tile with 'Account credentials are valid' message">}}
+
+5. Finally, click `Install integration`. For the purposes of this demonstration, all options have been selected (only `Collect custom metrics` is unticked by default):
```

Comment:
```suggestion
5. Finally, click `Install integration`. Select the services you want to use. For the purposes of this demonstration, all options have been selected (only `Collect custom metrics` is unticked by default):
```
I think the screenshot having all of the services selected is fine, but you'll want to be more specific about what you need for this getting started.